### PR TITLE
Code Coverage for integration tests scenarios

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

#220 

# Overview

- [ ] I have ran the wallet server in a separate thread. This way, we prevent the scenario from escaping code coverage :) 

# Comments

Note: I've also removed the 'Local' network target as it's not really a thing and isn't much use anyway.

### CODE REVIEW

**Comment by piotr-iohk** [comment]
>Note: I've also removed the 'Local' network target as it's not really a thing and isn't much use anyway.

So I suppose that the downside is that we no longer have a `cardano-wallet-launcher` being started as part of any tests on CI...

**Comment by KtorZ** [comment]
@piotr-iohk Indeed, but that's something we can re-enable fairly easily to make sure we still have some automated verification for this.

**Comment by KtorZ** [comment]
Should I maybe do it in this PR directly?

**Comment by piotr-iohk** [comment]
>Should I maybe do it in this PR directly?

sure, why not :)

**Review by piotr-iohk — APPROVED** [review]
lgtm :+1:  (as there are some discussions around `cardano-wallet-launcher`, it can be done in another PR if needed)

**Comment by KtorZ** [comment]
@piotr-iohk Added a "Cardano.LauncherSpec" which just call the launcher using the command-line, and wait for a bit. If the launcher crashes or doesn't start, the test fails.

File: lib/http-bridge/test/integration/Cardano/LauncherSpec.hs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                Inherit
+        handle <- async $ void $ launch [cardanoWalletLauncher]
+        let fiveSeconds = 5000000
+        winner <- race (threadDelay fiveSeconds) (wait handle)
+        case winner of
+            Left _ ->
+                cancel handle
+            Right _ ->
+                expectationFailure
+                    "cardano-wallet-launcher isn't supposed to terminates. \
```
Comment (piotr-iohk) [inline lib/http-bridge/test/integration/Cardano/LauncherSpec.hs]: ah, typo (`terminates`)

**Pull request merged**